tests/Replication2/ReplicatedLog/CalcCommitIndexTest.cpp

```diff
@@ -32,88 +33,108 @@ using namespace arangodb::replication2;
 using namespace arangodb::replication2::algorithms;
 using namespace arangodb::replication2::replicated_log;
 
-struct CalcCommitIndexTest : ::testing::Test {};
+struct CalcCommitIndexTest : ::testing::Test {
+
+  auto verifyQuorum(std::vector<ParticipantStateTuple> const& participants, std::vector<ParticipantId> const& quorum, LogIndex const& expectedLogIndex) {
+
+    // every member of the quorum needs to have at least the expectedLogIndex,
+    // must not be failed or excluded.
+    for(auto const& pid : quorum) {
+      auto p = std::find_if(std::begin(participants), std::end(participants), [pid](auto& pst) { return pst.id == pid; });
```

Comment:
You could add additional information using https://github.com/google/googletest/blob/master/docs/advanced.md#adding-traces-to-assertions